admin/common_auth/models.py

```diff
@@ -73,7 +73,14 @@ class Meta:
     #     return self.is_admin
 
     @property
+    def full_name(self):
```

Comment:
@samchrisinger There's no particular reason for overriding those methods to return email. Simply, I overrode them b/c our model doesn't use username. I can remove the override